### PR TITLE
Add Tech Radar review process

### PR DESCRIPTION
**Pull request by mwz**
This PR formalises the process for reviewing our Tech Radar, see the changes for more details.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +49,93 @@ effectiveness and knowledge sharing across teams and the Technology Radar is an
...
+any time to resume the process from the point it was decided to put that
+technology on Hold. The proposal should be supported by any evidence that makes
+the decision to put the technology on Hold no longer relevant, e.g. the
+technology has matured enough, and the main concerns have been addressed since
+it was last reviewed. The proposal will be reviewed, and a decision will be made
+by the HoE after considering the recommendation from the Tech Radar Advisory
+Group on whether the process can be resumed or the technology should remain on
+Hold.
+
+See the following flowchart for a sunmmary of the process:
```
Comment (chris-brindley) [inline README.md:123]:
```suggestion
See the following flowchart for a summary of the process:
```

File: README.md [hunk 2]
```diff
@@ -37,9 +37,10 @@ Each quadrant is divided into four rings:
   engineers in our organisation have used them before and will share knowledge
   and experiences.
 - **Hold**  
-  Technologies not recommended to be used for new projects. Technologies that we
-  think are not (yet) worth to (further) invest in. The technologies should not
-  be used for new projects, but usually can be continued for existing projects.
+   Technologies not recommended to be used for new projects. Technologies that we
+  think are not (yet) worth to (further) invest in. The technologies should not be
```
Comment (brad-allan-ovo) [inline README.md:41]:
```suggestion
  think are not (yet) mature enough to warrant continued investment. The technologies should not be
```

**Review by brad-allan-ovo — DISMISSED** [review]
Think it is worth adding the draw.io (assumption) file to the repo so flowcharts can be updated in the future.

**Comment by mwz** [comment]
> Think it is worth adding the draw.io (assumption) file to the repo so flowcharts can be updated in the future.

I've added it into Google Drive so it's easier to comment and collaborate, see https://github.com/ovotech/ovo-com-tech-radar/pull/13/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R126-R127. I'll add it to our team drive so that everyone has contributor access.

**Comment by brad-allan-ovo** [comment]
>  I've added it into Google Drive so it's easier to comment and collaborate, see https://github.com/ovotech/ovo-com-tech-radar/pull/13/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R126-R127. I'll add it to our team drive so that everyone has contributor access.

Worth adding this link in a .txt file next to the svg in the repo then?

**Comment by mwz** [comment]
> > I've added it into Google Drive so it's easier to comment and collaborate, see [#13 (files)](https://github.com/ovotech/ovo-com-tech-radar/pull/13/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R126-R127). I'll add it to our team drive so that everyone has contributor access.
> 
> Worth adding this link in a .txt file next to the svg in the repo then?

sure, done in 867bee6

**Review by brad-allan-ovo — DISMISSED** [review]

**Pull request merged**